### ISSUE TITLE
[Task] Extend "export API" feature to handle multiple aspect models models at once

### ISSUE THREAD

**Issue by Kostadin-Ivanov** [eclipse-esmf/esmf-sdk#453]
**Is your task related to a problem? Please describe.**
Currently we can export a single aspect model to OpenAPI. 
Could it be possible if, lets say we have a set of models which are connected to each other, either by direct connection, i.e. shared component or one model is member of other, or by sharing same namespace, to export a combined OpenAPI which includes all specified models or all models under specified namespace.

**Describe the solution you'd like**
In other words, could we have a functionality where we can pass list of aspect models or a namespace and to convert these into single OpenAPI?

**Describe alternatives you've considered**
N/A

**Additional context**
Not available at the moment, but I am happy to help in further discussion and ideas about implementation.

**Comment by chris-volk** [comment]
Since we this as a matter of style rather than a mandatory feature, this is not highest priority at the moment.
One API per Aspect fits the concept.

Could you post process by redoc / by link in one landing page (see https://redocly.com/blog/combining-openapis/) for the time being, @Kostadin-Ivanov , or is this a major obstacle in your setup?

**Comment by Kostadin-Ivanov** [comment]
Hello @chris-volk ,
not at all.

This option to generate OpenAPI for one or more - grouped aspect models, was mentioned in some of our "early stage" calls and if my memory works, I think that it came from @atextor  as a "nice to have" idea.

Anyway, as I mentioned this is not a blocker for us so feel free to postpone it.
